src/include/baselib/messaging/BrokerErrorCodes.h

```diff
@@ -90,24 +75,25 @@ namespace bl
                 return false;
             }
 
-            static auto tryGetExpectedErrorMessage( SAA_in const eh::error_code& ec ) NOEXCEPT -> const std::string&
+            static auto tryGetExpectedErrorMessage( SAA_in const eh::error_code& ec ) -> std::string
             {
                 if( ! isExpectedErrorCode( ec ) )
                 {
                     return bl::str::empty();
                 }
 
-                /*
-                 * The global error codes map will be initialized once on the first call
-                 */
-
-                static const auto& g_map = initExpectedErrorsGlobalMap();
-
-                const auto pos = g_map.find( ec.value() );
-
-                if( pos != g_map.end() )
+                switch( ec.value() )
                 {
-                    return pos -> second;
+                    case AuthorizationFailed:
+                        return ec.message();
+                    case ProtocolValidationFailed:
+                        return ec.message();
+                    case TargetPeerNotFound:
+                        return "The server is currently unavailable";
+                    case TargetPeerQueueFull:
+                        return "The server is too busy";
+                     default:
+                        return bl::str::empty();
                 }
 
                 return bl::str::empty();
```

Comment:
no need for this or for the default above

Reply:
Fixed.